packages/@aws-cdk/aws-cloudfront/lib/distribution.ts

```diff
@@ -187,7 +189,7 @@ export class Distribution extends Resource implements IDistribution {
     this.additionalBehaviors.push(new CacheBehavior(originId, { pathPattern, ...behaviorOptions }));
   }
 
-  private addOrigin(origin: IOrigin): string {
+  private addOrigin(origin: IOrigin, addingAFailoverOrigin: boolean = false): string {
```

Comment:
I like this slightly better, but don't feel super-strongly about it.

```suggestion
  private addOrigin(origin: IOrigin, isFailoverOrigin: boolean = false): string {
```

Reply:
Sure. Changed.